language/stdlib/modules/libra.move

```diff
@@ -262,6 +262,40 @@ module Libra {
         };
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // Treasury Compliant specific methods for DDs
+    ///////////////////////////////////////////////////////////////////////////
+
+    // Publish `preburn` under 'account'. Used for bootstrapping designated dealer
+    // as assocation TC account is creating this resource for DD
+    public fun publish_preburn_to_account<Token>(creator: &signer, account: &signer) {
+        assert_account_is_blessed(creator);
+        let preburn = Preburn<Token> { requests: Vector::empty(), is_approved: true };
```

Comment:
Unrelated, but I think we can kill the `is_approved` bool now that this is the only way to create a `Preburn` resource...

Reply:
OK, i think i'll do this in a follow up PR